features/step_definitions/email_steps.rb

```diff
@@ -46,150 +49,153 @@ def act_as_user(username)
 end
 
-Then /^(?:I|they|"([^"]*?)") should receive (an|no|\d+) emails? with subject "([^"]*?)"$/ do |address, amount, subject|
-  unread_emails_for(address).select { |m| m.subject =~ Regexp.new(Regexp.escape(subject)) }.size.should == parse_email_count(amount)
+Then "{email_address} should receive {amount} email(s) with subject {string}" do |address, amount, subject|
+  unread_emails_for(address).select { |m| m.subject =~ Regexp.new(Regexp.escape(subject)) }.size == amount
 end
 
-Then /^(?:I|they|"([^"]*?)") should receive (an|no|\d+) emails? with subject \/([^"]*?)\/$/ do |address, amount, subject|
-  unread_emails_for(address).select { |m| m.subject =~ Regexp.new(subject) }.size.should == parse_email_count(amount)
+Then "{email_address} should receive {amount} email(s) with subject {}" do |address, amount, subject|
+  unread_emails_for(address).select { |m| m.subject =~ Regexp.new(subject) }.size.should == amount
 end
 
-Then /^(?:I|they|"([^"]*?)") should receive an email with the following body:$/ do |address, expected_body|
-  open_email(address, :with_text => expected_body)
+Then "{email_address} should receive an email with the following body:" do |address, expected_body|
+  open_email(address, with_text: expected_body)
 end
 
 #
 # Accessing emails
 #
 
 # Opens the most recently received email
-When /^(?:I|they|"([^"]*?)") opens? the email$/ do |address|
+When "{email_address} open(s) the email" do |address|
   open_email(address)
 end
 
-When /^(?:I|they|"([^"]*?)") opens? the email with subject "([^"]*?)"$/ do |address, subject|
-  open_email(address, :with_subject => subject)
+When "{email_address} open(s) the email with subject {string}" do |address, subject|
+  open_email(address, with_subject: subject)
 end
 
-When /^(?:I|they|"([^"]*?)") opens? the email with subject \/([^"]*?)\/$/ do |address, subject|
-  open_email(address, :with_subject => Regexp.new(subject))
+When "{email_address} open(s) the email with subject {}" do |address, subject|
+  open_email(address, with_subject: Regexp.new(subject))
 end
 
-When /^(?:I|they|"([^"]*?)") opens? the email with text "([^"]*?)"$/ do |address, text|
-  open_email(address, :with_text => text)
+When "{email_address} open(s) the email with text {string}" do |address, text|
+  open_email(address, with_text: text)
 end
 
-When /^(?:I|they|"([^"]*?)") opens? the email with text \/([^"]*?)\/$/ do |address, text|
-  open_email(address, :with_text => Regexp.new(text))
+When "{email_address} open(s) the email with text {}" do |address, text|
+  open_email(address, with_text: Regexp.new(text))
 end
 
 #
 # Inspect the Email Contents
 #
 
-Then /^(?:I|they) should see "([^"]*?)" in the email subject$/ do |text|
+Then "I/they should see {string} in the email subject" do |text|
   current_email.should have_subject(text)
 end
 
-Then /^(?:I|they) should see \/([^"]*?)\/ in the email subject$/ do |text|
+Then "I/they should see {} in the email subject" do |text|
```

Comment:
IMO it will be more consistent to have `{string}` here but it is the original so maybe not a big issue.